bond/bond_test.go

```diff
@@ -315,7 +314,7 @@ var _ = Describe("tuning plugin", func() {
 			By("validating the macs are not duplicated")
 			slave1, err = netlink.LinkByName(Slave1)
 			Expect(err).NotTo(HaveOccurred())
-			slave2, err = netlink.LinkByName(Slave2)
+			slave2, _ = netlink.LinkByName(Slave2)
```

Comment:
Should we be doing this instead? why ignore the error?
```suggestion
			slave2, err= netlink.LinkByName(Slave2)
			Expect(err).NotTo(HaveOccurred())
```

Reply:
yes, it's better to check the error indeed. Thank you.